YOUR_ADMIN/includes/init_includes/init_bc_config_install_or_upgrade.php

```diff
@@ -247,6 +247,13 @@
         'configuration_title' => 'Header Nav Bar Link Color on Hover',
         'configuration_value' => '#cccccc',
         'sort_order' => 2120,
+    ],
+        'ZCA_HEADER_NAVBAR_LINK_BACKGROUND_COLOR_HOVER' => [
```

Comment:
This line needs to line up with the other configuration names (i.e. left 4 spaces).